Week-4/formForJoiningASocialNetwork.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Social Network</title>
+    <style>
```

Comment:
This is ok for now but going forward create seperate CSS file and then import the same in HTML.

Reply:
Sure.